### ISSUE TITLE
Trains in Deeprun Tram are phased.

### ISSUE THREAD

**Issue by Cabro** [elysium-project/server#3056]
The bug is like this:
- if you enter from IF side, you will only see the train on the track closer to the entrance, you will never see the one on the further track (sometimes you will see players levitating on what should be the train, but you don't see it).
- if you enter from SW side, you will only see the train on the track further from the entrance, you will never see the one on the closer track.
- if you are standing on the train, once it started moving, after about half of the distance you stop seeing NPCs/objects (you can't see Nessy), and you don't see the objects from the side of the arrival (such as benches/players/rats etc) until after you get off the train.

**Comment by Mostprovidedlinks** [comment]
Yeah when I board the tram Everything disapears and ruins the feel of the tram

**Comment by Mostprovidedlinks** [comment]
Yeah when I board the tram Everything disapears and ruins the feel of the tram

Even players around me
